bin/compute-gnocchi-classpath.sh

```diff
@@ -1,42 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Why is this file deleted?

Reply:
I wasn't using it, and didn't think there was a need for it. What is the typical use case of these?

Reply:
Generally we have used this file in the past to avoid redundancy when computing the classpath in shell and submit scripts. I would keep this file, as well as compute-gnocchi-jars, and call these in gnocchi-submit and gnocchi-shell scripts